object-search/requirements.txt

```diff
@@ -1 +1 @@
-jina[cv,http]==0.6.4
\ No newline at end of file
+jina[cv,http]==0.6.7
```

Comment:
we will need to make usre that we install the latest version in the `ci.yml` right?